### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by CodeEagle** [DroidsOnRoids/MPParallaxView#8]
Retain Cycle here:

`self` -> `motionManager` -> `self`

https://github.com/DroidsOnRoids/MPParallaxView/blob/master/MPParallaxView.swift#L104-L110

```swift
motionManager.startAccelerometerUpdates(to: currentQueue) {data, error in
                self.accelerometerMovement = AccelerometerMovement(x: data?.acceleration.x ?? 0.0, y: data?.acceleration.y ?? 0.0)
                UIView.animate(withDuration: 0.1, animations: {
                    self.applyParallaxEffectOnView(basedOnTouch: nil)
                    self.applyGlowEffectOnView(basedOnAccelerometerMovement: self.accelerometerMovement)
                }) 
            }
```

**Comment by michaello** [comment]
hey @CodeEagle ! 🙌
Good point, I will fix it soon - or maybe would you like to provide some small fix as PR? 🤓

**Comment by michaello** [comment]
Fixed by #9 .
